lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs

```diff
@@ -255,17 +255,17 @@
                     </div>
                 </dl>
           
-
-                <p>To access this sub-service, sign in to your DfE Sign-in account at <a href="<%=signInUrl%>">services.sign-in</a>. Or, you can select the button below to sign in to your account.</p>
-
+                <% if (isMultipleSubServicesReq) { %>
+                    <p>To access your sub-services, sign into your DfE Sign-in account at <a href="<%=signInUrl%>">services.sign-in</a>. Or, you can select the button below to sign in to your account.</p>
```

Comment:
please modify the content as per the protype 
![image](https://github.com/DFE-Digital/login.dfe.notifications.jobs/assets/70764046/1f89f6e6-d4e3-40ca-b715-cd6e1664579e)

Reply:
done